homeassistant/components/todo/strings.json

```diff
@@ -28,7 +28,7 @@
           "name": "Due date",
           "description": "The date the to-do item is expected to be completed."
         },
-        "due_date_time": {
+        "due_datetime": {
           "name": "Due date time",
```

Comment:
@c0ffeeca7 should we change the name of this parameter to `"Due datetime"`?

Reply:
good point. If we change it to something less technical anyway, and it does not have to include the underscore / match the code exactly, then we might as well change it to "Due date and time"?

Reply:
ah, sorry. I was confused. You were talking about the `name`.